script/RumpelConfig.sol

```diff
@@ -955,6 +965,18 @@ library RumpelConfig {
         return configs;
     }
 
+    function getPendleUSDEYTsTokenGuardConfigs() internal pure returns (TokenGuardConfig[] memory) {
+        TokenGuardConfig[] memory configs = new TokenGuardConfig[](1);
+
+        configs[0] = TokenGuardConfig({
+            token: MAINNET_PENDLE_YT_USDE_27MAR2025,
+            transferAllowState: RumpelGuard.AllowListState.ON,
+            approveAllowState: RumpelGuard.AllowListState.ON
```

Comment:
Note for most of the other YTs, we don't actually add `approve` to the allowlist (see `getFluidSusdeAndYTsTokenGuardConfigs` function as an example). The idea is to allow the minimal number of functions, and users don't need to deposit their YTs into anything (which would require `approve`), so we haven't added

Reply:
got it, makes sense
I changed to OFF for now